docs/_concepts/archive-authz.md

```diff
@@ -0,0 +1,96 @@
+2. Event type owners should not be able to accidentally forbid the archiving applications from reading events in an event type;
+3. Event type owners who do not wish for their events to be read by the archiving applications should get in touch with the administrators of the archiving applications;
+4. Event type owners should be made aware that the events published to their event type will be archived;
+5. Only administrators can update the list of archiving applications. This operation should be simple.
+
+The ability to exclude some event types from archival is not discussed in this concept.
+
+## Solution 1: add all archiving applications to the list of authorized event types
+
+The most simple solution is to add the archiving applications to the list of authorized event types:
+
+- If an event type does not have an authorization section, do nothing;
+- If an event type has an authorization section, add the list of archiving applications to the read section;
+- If an event type is updated and has an authorization section, make sure that the archiving applications cannot be removed from the read section.
+
+Nakadi will have to know the list of archiving applications; to do this, a new endpoint will be created:
+```
+POST /settings/resources/archivers
+```
+and its complement:
+```
+GET /settings/resources/archivers
+```
+
+These endpoints will be accessible to administrators with ADMIN rights only.
+
+If the list of authorized applications is updated, all event types with an authorization section will be updated accordingly
+
+### List of Tickets (to be created)
+
+- Create new endpoints
+- Update all event types with authorization when the list of archivers is updated;
+- Add archiving applications to new event types, if there is an authorization section;
+- Forbid updates of event types that remove archiving applications.
+
+### Pros
+
+- Relatively easy to implement: no changes to the authorization logic;
+- Makes it obvious which applications can read from the event type.
+
+### Cons
+
+- Confusing for users: they are not able to remove some applications from the readers, and there is nothing that marks them as archiving applications;
+- Updating the list of archiving applications could be cumbersome, as it may require updating a lot of event types.
+
+## Solution 2: a new resource for archiving applications
+
+In this solution, we create a new resource, `archive`, for which applications can only get READ permissions (write and admin must be empty).
+
+When a client tries to read from an event type with an authorization section, if the client is not in the list of clients authorized to read from the event type, Nakadi will check if the client is in the list of archiving applications.
+
+Nakadi will have to know the list of archiving applications; to do this, a new endpoint will be created:
+```
+POST /settings/resources/archivers
```

Comment:
What will be posted here? How the structure will look like?

Reply:
same as /settings/admins, but only for read (I'll update the concept)